src/modules/85-iacm/components/IACMPipelineSteps/IACMApproval/__tests__/StepMode.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { createRef } from 'react'
+import { render, act } from '@testing-library/react'
+
+import { StepViewType, StepFormikRef } from '@pipeline/components/AbstractSteps/Step'
+import { TestWrapper } from '@common/utils/testUtils'
+import StepMode from '../StepMode'
+
+describe('IACMApprovalStep', () => {
+  test('renders correctly', () => {
+    const { getByText } = render(
+      <TestWrapper>
+        <StepMode
+          stepViewType={StepViewType.Edit}
+          initialValues={{ timeout: '10s', name: '', identifier: '' }}
+          allowableTypes={[]}
+        />
+      </TestWrapper>
+    )
+
+    const nameField = getByText('Name')
+    const timeoutField = getByText('pipelineSteps.timeoutLabel')
+
+    expect(nameField).toBeInTheDocument()
+    expect(timeoutField).toBeInTheDocument()
```

Comment:
Again, not a problem, but since you're not doing anything else with these variables, you might as well inline them
```suggestion
    expect(getByText('Name')).toBeInTheDocument()
    expect(getByText('pipelineSteps.timeoutLabel')).toBeInTheDocument()
```

Will make zero difference, just an observation.